src/mantine-tiptap/src/RichTextEditor.tsx

```diff
@@ -50,7 +50,7 @@ const defaultProps: Partial<RichTextEditorProps> = {
   withTypographyStyles: true,
 };
 
-export function RichTextEditor(props: RichTextEditorProps) {
+export const RichTextEditor = forwardRef<HTMLInputElement, RichTextEditorProps>((props, ref) => {
```

Comment:
ref type is incorrect, it is div element, not input

Reply:
corrected, thanks

Reply:
thanks adarsha